app/views/paymentOptions.html

```diff
@@ -0,0 +1,18 @@
+<div class="sub-page-title-block">
+  <h1>Security Screen</h1>
+</div>
+<div class="sub-page-container">
+  <h2 class="text-center">As a security precaution, please enter your date of birth to access payment options for your ticket.</h2>
+  <div class="container-fluid topMargin alignContainerToPage">
+    <div class="row alignRowToPage">
+      <div class="col-sm-12 alignColumnsToPageToPage">
+        <div class="payment-options-date-picker">
+          <stl-courts-datepicker date-field="ctrl.dob" date-valid="ctrl.dobValid" date-over-18="ctrl.dobOver18" date-format="MM/DD/YYYY"></stl-courts-datepicker>
+          <div class="text-center">
+            <button type="button" class="btn btn-default" ng-click="ctrl.dobValidate()">View Payment Options</button>
```

Comment:
If the button is "View Payment Options", why is the function "dobValidate()" ? It's fine if DOB validation is part of the function but something more like "ctrl.viewPaymentOptions" or "ctrl.openPaymentOptions" would make more sense.